src/modules/functions.combined-fetcher/interface.nix

```diff
@@ -0,0 +1,10 @@
+{lib, ...}: let
+  l = lib // builtins;
+  t = l.types;
+in {
+  options = {
+    functions.fetchers.combinedFetcher = l.mkOption {
```

Comment:
I left it like that since I thought it would be obvious that it would be under "fetchers", but maybe changing the directory name is more correct? So make it `functions.fetchers.combined-fetcher`. It's fetcher related so do you think that would make more sense?

Reply:
I would keep it in the top-level. Because currently everything under `.fetchers` is a fetcher, and those two functions are not.